### PR TITLE
Updated deleting of user

### PR DESCRIPTION
**Pull request by zemuvier**
Now we can get the error if we try to delete non-existing user

### CODE REVIEW

File: my_dev_server/server/main.py [hunk 1]
```diff
@@ -69,9 +69,15 @@ def user_get(id):
...
     LOG.info('%s %s' % (request.method, id))
-    models.User.query.filter_by(id=id).delete()
-    base.session.commit()
-    return '200'
+    user = models.User.query.filter_by(id=id).first()
+    if user is None:
+        error_msg = "Can not delete user %s: it does't exist" % id
+        LOG.error(error_msg)
+        return error_msg
+    else:
```
Comment (esikachev) [inline my_dev_server/server/main.py]: you can make it without "else" block

File: my_dev_server/server/main.py [hunk 2]
```diff
@@ -69,6 +69,11 @@ def user_get(id):
 @mydev.route('/users/<id>', methods=['DELETE'], strict_slashes=False)
 def user_delete(id):
     LOG.info('%s %s' % (request.method, id))
+    user = models.User.query.filter_by(id=id).first()
+    if user is None:
+        error_msg = "Can not delete user %s: it does't exist" % id
+        LOG.error(error_msg)
+        return error_msg
```
Comment (IlyaMenkov) [inline my_dev_server/server/main.py]: we should return exception here

Comment (zemuvier) [inline my_dev_server/server/main.py]: Let's add this exception later, when we merge patch with all exceptions.

Comment (IlyaMenkov) [inline my_dev_server/server/main.py]: ok

**Pull request merged**